main/javascript-modifications/ses-guide.md

```diff
@@ -0,0 +1,505 @@
+# SES Guide
+
+This document is a guide to understanding *SES (Secure ECMAScript)*. In addition
```

Comment:
I'm afraid it is needed. Kate's mentioned that there's been feedback about duplication in the doc set. The problem is that the same person is likely to be two different audiences; first, someone getting started, who'll potentially want a good grounding in detail about what's going on and why (this doc) to best really understand things. But then they'll come back three months later and just want to find out as quickly as possible what options lockdown() has (for example). And that's what the SES-Reference doc will be. So we need to have a short introduction in both the Guide and the Reference explaining what each one is about and that the other one exists and why you might want to use it instead.

Reply:
No, this does need to be here. The problem is we've gotten feedback that some folk are bothered by duplication in the doc set. But some amount is necessary because the same person is likely to be two different audiences. When starting out, they'll hopefully want to read this, SES-Guide, which goes into detail about how and why SES is like it is to develop a good understanding of SES. But three months from now, they'll, for example, just want to quickly find what options go with lockdown(). Which is what the SES-Reference will be for; a stripped down version of the Guide with just "here's what this does and how to use it". So we need an intro paragraph in each explaining what this SES doc is for, and that the other's available if you want that version of SES documentation